specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2020-02-01-preview/managedservices.json

```diff
@@ -574,65 +574,65 @@
     "RegistrationDefinition": {
       "properties": {
         "properties": {
-          "description": "Properties of a registration definition.",
+          "description": "The properties of a registration definition.",
           "$ref": "#/definitions/RegistrationDefinitionProperties"
         },
         "plan": {
-          "description": "Plan details for the managed services.",
+          "description": "The details for the Managed Services offer’s plan in Azure Marketplace.",
           "$ref": "#/definitions/Plan"
         },
         "id": {
           "readOnly": true,
           "type": "string",
-          "description": "Fully qualified path of the registration definition."
+          "description": "The fully qualified path of the registration definition."
         },
         "type": {
           "readOnly": true,
           "type": "string",
-          "description": "Type of the resource."
+          "description": "The type of the Azure resource (Microsoft.ManagedServices/registrationDefinitions)."
         },
         "name": {
           "readOnly": true,
           "type": "string",
-          "description": "Name of the registration definition."
+          "description": "The name of the registration definition."
         }
       },
       "x-ms-azure-resource": true,
       "type": "object",
-      "description": "Registration definition."
+      "description": "The registration definition."
     },
     "RegistrationDefinitionProperties": {
       "properties": {
         "description": {
           "type": "string",
-          "description": "Description of the registration definition."
+          "description": "The description of the registration definition."
         },
         "authorizations": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/Authorization"
           },
-          "description": "Authorization tuple containing principal id of the user/security group or service principal and id of the build-in role."
+          "description": "The collection of principals in the managedBy tenant with permanent access to the managed tenant."
```

Comment:
>The collection of principals in the managedBy tenant with permanent access to the managed tenant [](http://example.com/codeflow?start=26&length=96)

The collection of principals in the managedBy tenant with permanent access during the delegation on the delegated resource.

We should convey that permanent access is only during the delegation period and also not to the whole managed tenant.

Reply:
Updated the wording a bit to convey the message